tests/examples/text/gears/string/linewrap.lua

```diff
@@ -0,0 +1,6 @@
+--DOC_GEN_OUTPUT --DOC_HIDE
+local gears = require("gears") --DOC_HIDE
+
+local test = "doit"
```

Comment:
```suggestion
local test = "do it"
```

Reply:
Realized it as soon as I went on family errands!